### DISCUSSION TITLE
how to add multiple educations/ fix formatting

### DISCUSSION THREAD

**Discussion by cvbate** [alshedivat/al-folio#2275] [Q&A] [answered]
I am trying to add multiple educations to the education section (MS, BA etc.) however on the localhost (docker) website theyre all bunched up and unreadable. Could anyone help me fix this? Thanks!
I was able to add multiple work experiences with no issue, but there's something about the formatting with the education section.....

ALSO, the second one is repeated twice because it is just a placeholder.

![image](https://github.com/alshedivat/al-folio/assets/122933733/560ecc29-a163-43cb-bc34-d29545c91979)

**Comment by george-gca** [comment, marked as answer]
You will basically need to play with the css of the page. I'd recommend using the developer tools available in [Chrome](https://developer.chrome.com/docs/devtools/overview/) or [Firefox](https://firefox-source-docs.mozilla.org/devtools-user/) to test the changes first, then apply the changes in the respective class in `_sass/_cv.scss`. There's actually an issue opened for this in #1882. But in your case I would suggest to also modify your dates, at least to display a shorter version of the month (like sep instead of september).
